docs/tutorials/plans-crowdloan-datahighway-kusama-tanganika.md

```diff
@@ -0,0 +1,226 @@
+---
+id: plans-crowdloan-datahighway-kusama-tanganika.md
+title: Crowdloan Plan for Tanganika Network (DataHighway's Kusama Parachain)
+sidebar_label: Crowdloan Plan for Tanganika Network (DataHighway's Kusama Parachain)
+---
+
+## About the Tanganika Network Crowdloan?
+
+To launch the Tanganika Network on Kusama we need help to bootstrap it. The project must secure enough funds to cover the cost of renting ("leasing") a parachain slot on the network. To meet this objective, the DataHighway team will run a crowdloan campaign on behalf of the network. For the Tanganika crowdloan, individuals agree to lock up a portion of their KSM in exchange for the DHX native network token. These tokens are locked from the point of participation until the slot is successfully won, and then through the entirety of the lease period (48 Weeks). A pool of **300,000 (300k) DHX** tokens representing **1%** of the DataHighway Treasury's Unlocked Reserves comprising 30,000,000 DHX tokens, which equates to **0.3%** of the DataHighway's network's genesis total supply will be distributed to crowdloan participants as a reward (proportionate to their share of the overall KSMs contributed to the crowdloan) throughout the lease period.
```

Comment:
Will there be early-bird incentives for people locking their KSM in the first days of the auction?
Will it be possible to increase the reward pool if it is observed that not enough KSM is obtained?

Reply:
> Will there be early-bird incentives for people locking their KSM in the first days of the auction?
no, not anticipated to be necessary
> Will it be possible to increase the reward pool if it is observed that not enough KSM is obtained?
no, not anticipated to be necessary